terraform/deployments/cluster-services/logging.tf

```
@@ -34,6 +34,21 @@ resource "helm_release" "filebeat" {
           "loadbalance" : true
           "ssl.enabled" : true
         }
+        "logging.files" : {
+          "keepfiles" : 7
+        }
+        "logging.metrics" : {
+          "enabled" : false
+        }
+        "http" : {
+          "enabled" : false
+        }
+        "logging" : {
+          "level" : "warn"
+        }
```

If I'm reading the [docs](https://www.elastic.co/guide/en/beats/filebeat/current/configuration-logging.html#configuration-logging) right, this should be a single key called `logging.level` rather than `level` nested inside `logging`.
```suggestion
        "logging.level" : "warn"
```

Similarly for most (but not all) of the others, e.g. `logging.metrics.enabled` should be all one key, but `output.file { enabled }` is already good.

It's super confusing that they use dots inside some of the keys. Where there really is nesting, it seems to be shown as nested in the docs (see logging.files for example) but it's not very clear 🙃